src/tls/openssl/tls_tcp.c

```diff
@@ -337,15 +337,12 @@ int tls_conn_change_cert(struct tls_conn *tc, const char *file)
 	r = SSL_use_certificate_file(tc->ssl, file, SSL_FILETYPE_PEM);
 #endif
 	if (r <= 0) {
-		DEBUG_WARNING("change cert: "
-			"cant't read certificate file: %s\n", file);
 		ERR_clear_error();
 		return EINVAL;
```

Comment:
```suggestion
		return ENOENT;
```
Maybe we should update the error code, too.